pkg/drivers/kic/kic.go

```diff
@@ -196,9 +197,26 @@ func (d *Driver) GetSSHHostname() (string, error) {
 
 // GetSSHPort returns port for use with ssh
 func (d *Driver) GetSSHPort() (int, error) {
-	p, err := oci.ForwardedPort(d.OCIBinary, d.MachineName, constants.SSHPort)
-	if err != nil {
-		return p, errors.Wrap(err, "get ssh host-port")
+	var perr error
+	var p int
+	findForward := func() error {
+		p, perr = oci.ForwardedPort(d.OCIBinary, d.MachineName, constants.SSHPort)
```

Comment:
Should this retry go into oci.ForwardedPort so that it can be useful to other use cases?

Reply:
I think this retry was actually useless, I will delete.

Reply:
removed the retry